package.json

```diff
@@ -27,14 +27,21 @@
   },
   "homepage": "https://github.com/silvermine/videojs-chromecast#readme",
   "dependencies": {
-    "class.extend": "0.9.1",
     "webcomponents.js": "git+https://git@github.com/webcomponents/webcomponentsjs.git#v0.7.24"
   },
   "devDependencies": {
+    "@babel/core": "^7.13.16",
```

Comment:
@kontrollanten could you pin these to exact versions using the `--save-exact` flag? Thanks

Reply:
By using package-lock.json file there should not be any reason to pin the dependencies in the package.json file. But I pinned them anyway just to be consistent. :)